streaming/index.js

```diff
@@ -638,6 +679,12 @@ const startServer = async () => {
         return;
       }
 
+      const localPayload = payload.account.username === payload.account.acct;
+      if (localPayload ? filterLocal : filterRemote) {
+        log.debug(`Message ${payload.id} filtered by feed settings`);
+        return;
+      }
+
```

Comment:
Should this maybe moved after the comment that is directly below? It says that the following code is about filtering and this *is* filtering.